doc/releases/release-notes-2.4.rst

```diff
@@ -501,6 +501,14 @@ Drivers and Sensors
     * :option:`CONFIG_TRACING_BACKEND_USB`
     * :option:`CONFIG_USB_UART_CONSOLE`
 
+  * CDC ACM Class was slightly reworked. The support for resume and
+    suspend events has been corrected. USB device support has got
+    its own work queue which is used by CDC ACM class by default.
```

Comment:
Those are two separate things + HID was updated in terms of Suspend/Resume as well.

```suggestion
  * Suspend/Resume event handling corrected for CDC ACM and HID.
  * Optional workqueue for USB subsystem introduced. Used by the
     CDC_ACM class by default.
```